cfn/paws-collector.template

```diff
@@ -685,6 +685,25 @@
          "Properties":{
             "VisibilityTimeout": 900,
             "MessageRetentionPeriod": 1209600,
+            "RedrivePolicy": {
+                "deadLetterTargetArn": {
+                    "Fn::Join": [
+                        "",
+                        [
+                            "arn:aws:sqs:",
+                            {
+                                "Ref": "AWS::Region"
+                            },
+                            ":",
+                            {
+                                "Ref": "AWS::AccountId"
+                            },
+                            ":Paws-shared-deadLetterQueue"
```

Comment:
THe DLQ should exist prior to using it here. Please update the https://algithub.pd.alertlogic.net/defender/paws-collectors-deploy-pipeline/blob/master/cfn/paws-shared-resources.template

Reply:
If we use same DLQ for all collectors can we differentiate DLQ message to identify which collector a particular DLQ message belongs to?

Reply:
Added extra param to identify the message from which collector in case it failed and went in DLQ.